File: Python/Funciones de Alto orden/Ejemplo1.py
__author__ = 'AlumnoT'
def porDos(n):
    return n*2

def multiporDos(lista):
    return map(porDos,lista)
#main
print multiporDos([1,4,5,2,3,10])

print map(lambda a:a*2,[1,2,3,4])

File: AccesoADatos/XPATH sin abreviar/src/xpath/sin/abreviar/Ejemplo1.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package xpath.sin.abreviar;

/**
 *
 * @author AlumnoT
 */
public class Ejemplo1 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        //herramientas.Herramientas.consultasXPATH("libro.xml", "/child::libro/child::precio[self::precio='3.50'/parent::libro/child::titulo");
        herramientas.Herramientas.consultasXPATH("libro.xml", "/child::libro/child::precio");
    }
    
}
